tests/backup/backup_rancher_test.go

```diff
@@ -1,6 +1,192 @@
 package tests
 
 import (
+	"fmt"
+	. "github.com/onsi/ginkgo"
+	"github.com/pborman/uuid"
+	"github.com/portworx/sched-ops/k8s/core"
+	"github.com/portworx/torpedo/drivers/backup"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/drivers/scheduler/rke"
+	"github.com/portworx/torpedo/pkg/log"
+	. "github.com/portworx/torpedo/tests"
 	_ "github.com/rancher/norman/clientbase"
 	_ "github.com/rancher/rancher/pkg/client/generated/management/v3"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
+
+// This testcase takes backup of single namespace and restore to namespace in same and different project
+var _ = Describe("{SingleNamespaceBackupRestoreToNamespaceInSameAndDifferentProject}", func() {
```

Comment:
If possible, please make corrections to the step descriptions or log messages, if there are any.

Reply:
done